src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,75 @@
+import AddIcon from '@mui/icons-material/AddCircleOutlineOutlined';
+import ListIcon from '@mui/icons-material/FormatListBulletedOutlined';
+import LogoutIcon from '@mui/icons-material/LogoutOutlined';
+import ShareIcon from '@mui/icons-material/ShareOutlined';
+import { IconButton, Tooltip } from '@mui/material';
+import { Link } from 'react-router-dom';
+import Title from '../../assets/smartshopperr_title_transparent.png';
+import './Navigation.css';
+
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+
+export const Navigation = ({ setToken, token }) => {
+  const navigate = useNavigate();
+
+  const handleSignOut = () => {
+    setToken('');
+    localStorage.removeItem('token');
+    navigate('/');
+  };
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(token).then(() =>
+      toast(
+        ({ closeButton }) => (
+          <div>
+            Token copied to clipboard! <br /> <strong>{token}</strong>
+          </div>
+        ),
+        {
+          autoClose: 5000,
+          hideProgressBar: true,
+          pauseOnHover: true,
+          draggable: false,
+          closeOnClick: false,
+        },
+      ),
+    );
+  };
+
+  const navIconStyle = {
+    fontSize: '2.5rem',
+    display: 'inline-block',
+  };
```

Comment:
Would it be possible for the icons to be black instead of gray?

Reply:
Good call, just made that change on the final commit :)